docs/index.asciidoc

```diff
@@ -267,6 +267,17 @@ fields => {
 
 List of elasticsearch hosts to use for querying.
 
+This option can accept an environment variable containing one or more hostnames separated by whitespace.
+Strings separated by whitespace are treated as separate entries.
+
+*Examples:*
+
+* `ES_HOSTS="es1.example.com es2.example.com:9201 es3.example.com:9201"`
+* `ES_HOSTS="127.0.0.1:9200 127.0.0.2:9200"`
+* `ES_HOSTS="http://127.0.0.1 http://127.0.0.2"`
```

Comment:
not sure what the back-story is but these aren't working properly.
the first 2 examples will start although I do not think the host fall-over is working (only the first entry is extracted by the client library). for the second half of examples - they simply fail to start the plugin.